### PR TITLE
Feature/typescript

### PR DESCRIPTION
**Pull request by FrazierLE**
#### What does this PR do?
Adds TypeScript to App.js and to apiCalls.js.

#### Where should your reviewer start
You can either start at apiCalls.js and walk through it that way or start in App.js and work their way to apiCalls.js

#### What (if any) features are you implementing
Added TypeScript to two files.

#### What (if anything) did you refactor?
I refactored Router in JSX. Router version 6 doesn't use exact anymore. 

#### Were there any issues that arose?
Just still learning TypeScript

#### Is there anything that you need from your teammate?
If they have any ideas about a more specific type for the Promise type in apiCalls.js

### CODE REVIEW

**Comment by EvanSSwanson** [comment]
Did the review together, looks good.

File: src/Components/App/App.tsx [hunk 1]
```diff
@@ -16,40 +12,42 @@ interface Guesses {
 }
 
 const App: React.FC = () => {
-  const [data, setData] = useState<countryData>()
+  const [data, setData] = useState<string[]>([])
```
Comment (sertmer) [inline src/Components/App/App.tsx:15]: Can you think of a more descriptive variable than "data"? What does this data represent?

File: src/Components/App/App.tsx [hunk 2]
```diff
@@ -16,40 +12,42 @@ interface Guesses {
...
     console.log("data", data)
-    keepScore('Asia')
   }, [])
 
-  const keepScore = (continent: string) => {
+  const keepScore = (continent: keyof Guesses): string  => {
     const total = correctGuesses[continent] + incorrectGuesses[continent]
-    const score = (correctGuesses[continent] / total * 100).toFixed() + '%'
-    console.log('score', score)
+    const score = (correctGuesses[continent]/total * 100).toFixed() + '%'
```
Comment (sertmer) [inline src/Components/App/App.tsx:37]: Is score something we should potentially be tracking in state?

File: src/apicalls.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const getData = () => {
+const getData = (): Promise<any> => {
```
Comment (sertmer) [inline src/apicalls.tsx:1]: Can you think of a more descriptive function name? What kind of data is it?

**Review by sertmer — COMMENTED** [review]
Left a couple questions for yall!

**Pull request merged**